src/pages/BookDetailsPage/ReviewList/ReviewList.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, Divider, Typography } from '@mui/material'
+import style from './ReviewList.module.css'
+import { ReviewComponent } from './ReviewComponent'
+import reviewStar from './../../../assets/totalReviewStar.svg'
+import { useEffect, useState } from 'react'
+import { ReviewForm } from '../../ReviewForm'
+
+interface User {
+  id: string
+  fullName: string
+  email: string
+  imageUrl: string
+  role: string
+}
+
+interface Review {
```

Comment:
Review interface is double in ReviewComponent and ReviewList so please move it to types file and reuse the same type for multiple components.